flags/conditions.py

```diff
@@ -96,7 +95,8 @@ def parameter_condition(param_name, request=None, **kwargs):
         raise RequiredForCondition("request is required for condition "
                                    "'parameter'")
 
-    return request.GET.get(param_name) == 'True'
+    return param_name in request.GET
+    # return request.GET.get(param_name, '').lower() == 'true'
```

Comment:
Did you mean to leave this commented-out line here?

Reply:
Nope!